tests/functional/run-debug/expected/expected-stdout

```diff
@@ -0,0 +1,16 @@
+[%s] DEBUG: Component initialization completed [] []
+[%s] DEBUG: Sklik API request: client.loginByToken, json: ["0x2ff1bf323b7c8cffeff657e3ea25523cf5cbd18b660e206a4dbe568627c55b8481acd-sklik@keboola.com"] [] []
```

Comment:
nesmíme nikde logovat tokeny! to musí pryč

Reply:
opraveno